src/server/web_server.js

```diff
@@ -295,11 +295,28 @@ app.get('/metrics/nsfs_stats', async (req, res) => {
 });
 
 function _create_nsfs_report() {
-    const nsfs_counters = stats_aggregator.get_nsfs_stats();
     let nsfs_report = '';
+
+    const nsfs_counters = stats_aggregator.get_nsfs_io_stats();
+    // Building the report per io and value
     for (const [key, value] of Object.entries(nsfs_counters)) {
         nsfs_report += `NooBaa_nsfs_${key}: ${value}<br>`;
     }
+
+    const op_stats = stats_aggregator.get_op_stats();
+    // Building the report per op name key and value
+    for (const [op_name, obj] of Object.entries(op_stats)) {
+        nsfs_report += `<br>`;
+        for (const [key, value] of Object.entries(obj)) {
+            nsfs_report += `NooBaa_nsfs_${op_name}_${key}: ${value}<br>`;
+        }
+    }
+
+    //TODO: remove the LMLM. 
+    // Do we want to log the report as it will be gone on the second call? 
+    // Do want it log0 or maybe log1 (Personally I think log0)?  
+    dbg.log0(`LMLM: _create_nsfs_report: nsfs_report ${nsfs_report}`);
```

Comment:
leftover

Reply:
@nimrod-becker @romayalon WDYT? 
Should we keep this log?

Reply:
After talking to @romayalon leaving the log and changing it to 1